File: Beginner/1046 - Game Time/1046.c
#include <stdio.h>
#include <math.h>

int main(){

    int A, B;
    scanf("%i %i", &A, &B);
    if(A>=B)
        B+=24;

    printf("O JOGO DUROU %i HORA(S)\n", B-A);

    return 0;
}

File: Beginner/1046 - Game Time/1046.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        string[] input = Console.ReadLine().Split(' ');
        int start = int.Parse(input[0]);
        int end = int.Parse(input[0]);

        if (end < start)
            end += 24;
        Console.WriteLine($"O JOGO DUROU {end - start} HORA(S)");
    }
}